src/main/java/uk/ac/ebi/eva/pipeline/Application.java

```diff
@@ -34,6 +34,7 @@
 public class Application {
 
     private static final Logger log = LoggerFactory.getLogger(Application.class);
+    public static final String VARIANT_ANNOTATION_MONGO_PROFILE = "variant-annotation-mongo";
```

Comment:
I don't know if it will be interesting at some point mixing databases or just because we need to transfer data, simply i thought it's not a constraint that we need to add explicitly now. Anyway i think profiles also let you enable secondary profiles like if you put profile mongo-all adss profiles mongoA and mongoB. I'll return to you when i find the related documentation.